### PR TITLE
hooks: sphinx: update

### PR DESCRIPTION
**Pull request by xoviat**

### CODE REVIEW

File: PyInstaller/hooks/hook-sphinx.py [hunk 1]
```diff
@@ -73,8 +73,12 @@
...
-if is_module_satisfies('sphinx >= 1.3.1') and is_module_satisfies('sphinx < 1.4'):
+if is_module_satisfies('sphinx >= 1.3.1') and is_module_satisfies(
+        'sphinx < 1.4'):
     datas.extend(collect_data_files('alabaster'))
     datas.extend(collect_data_files('sphinx_rtd_theme'))
 elif is_module_satisfies('sphinx >= 1.3.1'):
     datas.extend(collect_data_files('alabaster'))
+
+if is_module_satisfies('sphinx >= 1.6.1'):
+    hiddenimports += collect_submodules('sphinx.environment.collectors')
```
Comment (xoviat) [inline PyInstaller/hooks/hook-sphinx.py]: These two lines are the only non-formatting change.

**Comment by xoviat** [comment]
@htgoebel please merge this soon so that we don't have failing tests

File: PyInstaller/hooks/hook-sphinx.py (hunk 1 above)
Comment (htgoebel) [inline PyInstaller/hooks/hook-sphinx.py]: So please split this in a functional and a non-formatting commit. I'd even prefer not changing the formatting at all. It only creates a huge diff without real use.

Comment (xoviat) [inline PyInstaller/hooks/hook-sphinx.py]: @htgoebel Done.

Comment (htgoebel) [inline PyInstaller/hooks/hook-sphinx.py]: 👍 That's been quick :-) But what should the commit massage "yapf" mean? Please use something less cryptic :-)

Comment (xoviat) [inline PyInstaller/hooks/hook-sphinx.py]: Googling yapf will reveal the answer.

**Pull request closed without merging**